build_tools/cmake/iree_benchmark_suite.cmake

```diff
@@ -80,6 +80,14 @@ function(iree_benchmark_suite)
 
   iree_package_name(PACKAGE_NAME)
 
+  # Add the benchmark suite target for a specific platform.
+  cmake_path(GET CMAKE_CURRENT_LIST_FILE FILENAME _PLATFORM_NAME)
+  cmake_path(REMOVE_EXTENSION _PLATFORM_NAME)
+  set(SUITE_SUB_TARGET "iree-benchmark-suites-${_PLATFORM_NAME}")
```

Comment:
I'd prefer to pass the platform name, or more generally the "suite name", explicitly as an argument, rather than connect it to the file name. That way we can:
* generate multiple test suites in a single file (e.g. x86, x86_with_flags)
* have multiple files contribute to the same test suite
* have benchmarks that are independent of target platform still use this infrastructure, without _having_ to group them into a "generic.cmake" (for example)

Reply:
Done.